play-media-native/screens/EditAthleteDetails.tsx

```diff
@@ -0,0 +1,223 @@
+import { getAccentColor } from "../helpers/colorHelper";
+import { getDate, getYear } from "../helpers/dateHelper";
+import { theme } from "../theme/theme";
+import DateTimePicker from "@react-native-community/datetimepicker";
+import { Athlete } from "../interfaces/athlete";
+import { AthleteImages } from "../features/Screens/AthleteImages";
+
+const styles = StyleSheet.create({
+  container: {
+    backgroundColor: theme.colors.black.darkest,
+    paddingTop: theme.spacing.md,
+    paddingHorizontal: theme.spacing.sm,
+  },
+  button: {
+    position: "absolute",
+    right: -theme.spacing.sm,
+    top: -theme.spacing.xs,
+  },
+  label: {
+    color: theme.colors.white.DEFAULT,
+    marginBottom: theme.spacing.xxs,
+  },
+  item: {
+    marginBottom: theme.spacing.xs,
+    fontFamily: theme.fontFamily.bold,
+  },
+  textInput: {
+    height: 37,
+    backgroundColor: theme.colors.white.DEFAULT,
+    padding: theme.spacing.xs,
+    marginBottom: theme.spacing.xs,
+  },
+  imageContainer: {
+    marginBottom: theme.spacing.md,
+  },
+  imageLabel: {
+    color: theme.colors.gray.DEFAULT,
+    marginBottom: theme.spacing.xs,
+  },
+  imageItem: { height: 300, width: "100%", marginTop: theme.spacing.xs },
+  imageGrid: {
+    marginTop: theme.spacing.xs,
+  },
+});
+
+export const EditAthleteDetailsScreen = ({ route, navigation }) => {
+  const [name, handleChangeName] = useState("");
+  const [nationality, handleChangeNationality] = useState("");
+  const [quote, handleChangeQuote] = useState("");
+  const [birthDate, handleChangeBirthDate] = useState(new Date());
+  const [careerStartDate, handleChangeCareerStartDate] = useState(new Date());
+  const [hobby, handleChangeHobby] = useState("");
+
+  const [showBirthDatePicker, setShowBirthDatePicker] = useState(false);
+  const [showCareerStartDatePicker, setShowCareerStartDatePicker] =
+    useState(false);
+
+  const setAthleteData = (athlete: Partial<Athlete>) => {
+    handleChangeName(athlete.athleteName);
+    handleChangeNationality(athlete.nationality);
+    handleChangeQuote(athlete.athleteQuote);
+    handleChangeBirthDate(athlete.dateOfBirth);
+    handleChangeCareerStartDate(athlete.careerStartDate);
+    handleChangeHobby(athlete.hobby);
+  };
+
+  const { data, isFetching } = useQuery(
+    "editathlete",
+    () => getAthleteById(route.params.id),
+    { onSuccess: (data) => setAthleteData(data.athlete) }
+  );
+  const athlete = data?.athlete;
+
+  useEffect(() => {
+    navigation.setOptions({
+      title: route.params.title,
+    });
+  }, []);
+
+  if (isFetching) {
+    return <LoadingScreen />;
+  }
+
+  if (!athlete) {
+    return <Text>Athlete could not be fetched!</Text>;
+  }
+
+  // TODO
+  const handleChangeSport = (id: string, title: string) => {
+    navigation.navigate("EditSport", {
+      id,
+      title,
+    });
+  };
+
+  // TODO
+  const handleAddMedia = (id: string, title: string) => {
+    navigation.navigate("AddMedia", {
```

Comment:
Is this TODO or does it work? For me it sometimes opens the screen, sometimes not... 😄